src/components/IEEEFunctionInput.vue

```diff
@@ -0,0 +1,108 @@
+<template>
+  <div class="function-input">
```

Comment:
Could we rename this file to `IEEEFormatInput`. I feel like calling it "function" in a programming context is going to cause confusion ;)